docker/serverless/gcloud_build_image

```diff
@@ -100,8 +100,9 @@ EOF
 
 NEW_IMAGE="gcr.io/${PROJECT}/endpoints-runtime-serverless:${ESP_FULL_VERSION}-${SERVICE}-${CONFIG_ID}"
 gcloud builds submit --tag "${NEW_IMAGE}" . --project="${PROJECT}"
+)
 
 # Delete the temporary directory we created earlier.
+# Move back to the previous directory with an echo.
 rm -r "${PWD}"
-)
-
+cd ~-
```

Comment:
Hmm,  if we enlarge () to: before line 77 to after line 108,   we don't need such "cd" back.   I believe () scope will restore to original folder, is that right

Reply:
Yes, but then you end up with extra output at the end when you `set +x` before `rm -f`

```
c2f9790b-3d85-4418-b086-2b324a9e53f0  2020-08-11T17:03:24+00:00  1M9S      gs://nareddyt-cloud-esf_cloudbuild/source/1597165403.568198-c15932054a884012813eb22745228e82.tgz  gcr.io/nareddyt-cloud-esf/endpoints-runtime-serverless:2.15.0-espv2-path-matcher-lrnm3ejpeq-uc.a.run.app-2020-05-13r1  SUCCESS
+ set +x
```

Preferably I'd like the last output to be the image name, nothing else.

Reply:
then, can we use pushd and popd?

Reply:
Worried about portability. Most articles I've read online mention that `cd -` is preferred over pushd/popd.

https://mamchenkov.net/wordpress/2017/03/22/pushdpopd-vs-cd/

> pushd and popd, it turns out, are built into Bash, but they are not standard POSIX commands, so not all the shells have them. pushd and popd, it turns out, are built into Bash, but they are not standard POSIX commands, so not all the shells have them. 

Given we'd had issues with this script on macos before (#258), i'd prefer to use `cd -`.

Reply:
OK,  thanks.